BoxContentSDK/BoxContentSDK/Requests/BOXCollectionItemsRequest.m

```diff
@@ -38,6 +38,8 @@ - (BOXAPIOperation *)createOperation
     
     if (self.requestAllItemFields) {
         parameters[BOXAPIParameterKeyFields] = [self fullItemFieldsParameterString];
+    } else if (self.requestAllItemFieldsAndExcludingFields) {
```

Comment:
so if we change the meaning of the array (to not also be it's own flag), then this moves out of the if block.